### PR TITLE
Adding a flag to skip plugin upload in cdap sandbox.

### PR DESCRIPTION
**Pull request by Vipinofficial11**
Description : 

Added a new flag ```skipPluginUpload``` to skip building plugin and uploading it on cdap sandbox, this flag can be used in any module or repository which is not a plugin repository (doesn't create Jar and Json file of plugin) but contains e2e tests.

### CODE REVIEW

File: src/main/scripts/run_e2e_test.py [hunk 1]
```diff
@@ -169,6 +173,7 @@ def run_shell_command(cmd):
     if module_to_build:
         if testrunner_to_run:
             print("TestRunner to run : " + testrunner_to_run)
+
```
Comment (itsankit-google) [inline src/main/scripts/run_e2e_test.py]: nit: remove extra empty line.

**Review by itsankit-google — APPROVED** [review]

File: src/main/scripts/run_e2e_test.py (hunk 1 above)
Comment (Vipinofficial11) [reply src/main/scripts/run_e2e_test.py]: Removed the extra line.

File: src/main/scripts/run_e2e_test.py [hunk 2]
```diff
@@ -79,68 +80,71 @@ def run_shell_command(cmd):
...
+                artifact_version = driver_details['modules'][module_to_build]['artifact_version']
+                get_driver_jar = requests.get(url)
+                assert get_driver_jar.ok or print(get_driver_jar.text)
+                driverData = get_driver_jar.content
+                print(f"Installing {artifact_name} driver")
+                res=requests.post(f"http://localhost:11015/v3/namespaces/default/artifacts/{artifact_name}", headers={"Content-Type": "application/octet-stream", "Artifact-Version": artifact_version, "Artifact-Plugins": driver_prop }, data=driverData)
+                assert res.ok or print(res.text)
+        else:
+            os.chdir("../..")
+
```
Comment (itsankit-google) [inline src/main/scripts/run_e2e_test.py]: nit: remove extra empty line.

Comment (Vipinofficial11) [reply src/main/scripts/run_e2e_test.py]: removed few extra lines [646f5fdf7de8c4ed6653656036d87424573b7062]

**Review by itsankit-google — COMMENTED** [review]
Please squash commits before merge.

**Comment by Vipinofficial11** [comment]
> Please squash commits before merge.

Squashed the commits.

**Comment by Vipinofficial11** [comment]
The build is passed, squashed the commits. Merging the PR.

**Pull request merged**